specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-03-01/orbital.json

```diff
@@ -0,0 +1,2228 @@
+          "Ground Station"
+        ],
+        "operationId": "AvailableGroundStations_Get",
+        "description": "Gets the specified  available ground station",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/groundStationNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting ground station resource.",
+            "schema": {
+              "$ref": "#/definitions/AvailableGroundStation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get GroundStation": {
+            "$ref": "./examples/AvailableGroundStationGet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Orbital/locations/{location}/operationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "OperationResults"
+        ],
+        "description": "Returns operation results.",
+        "operationId": "OperationsResults_Get",
+        "x-ms-examples": {
+          "KustoOperationResultsGet": {
+            "$ref": "./examples/OperationResultsGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the operation result.",
+            "schema": {
+              "$ref": "#/definitions/OperationResult"
+            }
+          },
+          "202": {
+            "description": "The operation is still in progress.",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "URL for determining when an operation has completed."
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Can you confirm it is true that this API is a long running operation? If it is, please specify x-ms-long-running-operation-options as you do for the other LRO APIs. To be honest, never see a get LRO before. LRO is designed for put/post/delete/patch. Besides, please https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation for more details.

Reply:
> 

@raych1 Do we allow a get API to be async (long running operation)?